public/views/backup.html

```diff
@@ -75,9 +75,7 @@
     <div class="row enable_text_select" ng-show="show">
         <div class="small-centered p10t p10b large-centered medium-centered large-8 medium-8 small-10 columns enable_text_select" style="border:1px solid gray; background:#eee;
 ">
-          <span ng-repeat="word in wordsC.mnemonicWords">
-            {{word}} 
-          </span> 
+          <span ng-repeat="word in wordsC.mnemonicWords"><span style="white-space:nowrap">{{word}}</span><span ng-show="index.defaultLanguageIsoCode == 'ja'">&#x3000;</span> </span>  
```

Comment:
Right, this is only to be shown to the user.

If the user copy and paste it (not recommended anyways) there will be
double spaces, but that will be handled by the normalization at the input.

It is needed to add the normal space (not to remove it as you suggested for
non JA languages) because otherwise all the 12 words appear in one line.



On Tue, Sep 15, 2015 at 1:24 PM, dabura667 <notifications@github.com> wrote:

> In public/views/backup.html
> <https://github.com/bitpay/copay/pull/3171#discussion_r39531674>:
>
> > @@ -75,9 +75,7 @@
> >      <div class="row enable_text_select" ng-show="show">
> >          <div class="small-centered p10t p10b large-centered medium-centered large-8 medium-8 small-10 columns enable_text_select" style="border:1px solid gray; background:#eee;
> >  ">
> > -          <span ng-repeat="word in wordsC.mnemonicWords">
> > -            {{word}}
> > -          </span>
> > +          <span ng-repeat="word in wordsC.mnemonicWords"><span style="white-space:nowrap">{{word}}</span><span ng-show="index.defaultLanguageIsoCode == 'ja'">&#x3000;</span> </span>
>
> Maybe
> <span ng-repeat="word in wordsC.mnemonicWords"><span
> style="white-space:nowrap">{{word}}</span><span
> ng-show="index.defaultLanguageIsoCode == 'ja'">&#x3000;</span><span
> ng-show="index.defaultLanguageIsoCode != 'ja'"> </span></span>
> or is there a more elegant way to toggle like this?
>
> btw, this is not what is being hashed, correct?
>
> phrase to be hashed should only contain one space between each word
> (\u3000 and normal space are both the same after NFKD normalization, so it
> does not matter which one is used, however it must be only 1 space.)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitpay/copay/pull/3171/files#r39531674>.
>



-- 
Matías Alejo Garcia
@ematiu
Roads? Where we're going, we don't need roads!